lib/flutter_map.dart

```diff
@@ -136,5 +136,6 @@ class MapPosition {
   final LatLng center;
   final LatLngBounds bounds;
   final double zoom;
-  MapPosition({this.center, this.bounds, this.zoom});
+  final bool hasGesture;
+  MapPosition({this.center, this.bounds, this.zoom, this.hasGesture = false});
```

Comment:
a MapPosition is not an event, so I would prefer that the `onPositionChanged` stream emits a new object; `MapPositionChangedEvent` with two fields; the MapPosition and a bool representing whether or not the map has moved due to the current user.

Reply:
ok now MapPositionChangedEvent emit two events